tests/common.rs

```diff
@@ -56,7 +56,12 @@ async fn wait_n_finalized_blocks_from(n: usize, url: &str) {
 	let mut interval = tokio::time::interval(Duration::from_secs(6));
 
 	loop {
-		if let Ok(block) = get_finalized_head::<Block, _>(url).await {
+		let rpc_service = RpcService::new(url, false).await;
```

Comment:
match/if let Ok here would be nicer here

Reply:
I don't like such nesting, but I guess this will be cleaner anyway